src/styles/blocks/about.scss

```diff
@@ -0,0 +1,40 @@
+.about {
+  &__title {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 100%;
+    letter-spacing: -1.5px;
+    margin-bottom: 32px;
+
+    @include onTablet {
+      font-size: 40px;
+    }
+
+    @include onDesktop {
+      font-size: 48px;
+    }
+  }
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 140%;
+    margin-bottom: 40px;
+
+    &-item {
+      margin-bottom: 16px;
+
+      &:nth-last-child {
+        margin-bottom: 0;
+      }
```

Comment:
Useless. Your list's bottom margin overrides this value with 40px.
Also, it doesn't work cause you didn't specify the argument - https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-last-child.